docs/components/camera/ffmpeg.md

```diff
@@ -1,15 +1,17 @@
 ---
-title: "Configure a FFmpeg Camera"
+title: "Configure an ffmpeg Camera"
 linkTitle: "ffmpeg"
 weight: 30
 type: "docs"
-description: "Uses a camera, a video file, or a stream as a camera."
+description: "Uses a camera device, a video file, or a stream as a camera."
 images: ["/icons/components/camera.svg"]
 tags: ["camera", "components"]
 # SMEs: Bijan, vision team
 ---
 
-A `ffmpeg` camera uses a camera, a video file, or a stream as a camera.
+The `ffmpeg` camera model uses a camera device, a video file, or a stream as a camera.
+
+When used with a streaming camera, the `ffmpeg` camera model supports any streaming camera format that is supported by the [`ffmpeg` program](https://ffmpeg.org/), including MJPEG, H264, and MP4.
```

Comment:
Q: Is this fair to say then? "supports any format supported by `ffmpeg` program" ?

Reply:
yea, whatever the ffmpeg version they have installed supports, they will able to support